airflow/api_connexion/openapi/v1.yaml

```
@@ -1490,6 +1490,125 @@ paths:
         '404':
           $ref: '#/components/responses/NotFound'
 
+  /auth-info:
+    get:
+      summary: Get site authentication information
+      x-openapi-router-controller: airflow.api_connexion.endpoints.auth_endpoint
+      operationId: get_auth_info
+      tags: [Authentication]
+      responses:
+        '200':
+          description: Success.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/AuthInfo'
+
+  /auth-dblogin:
```

```suggestion
  /auth/login:
```

Since this works for DB and LDAP